openslides/core/config.py

```diff
@@ -144,7 +145,7 @@ def __setitem__(self, key: str, value: Any) -> None:
             for entry in value:
                 if not isinstance(entry, dict):
                     raise ConfigError(_('Every value has to be a dict, not {}.'.format(type(entry))))
-                whitelist = (
+                whitelist = (  # type: ignore
```

Comment:
Type is set, not?

Reply:
Mypy has a conflict with the `whitelist` in line 131. There it is a `Tuple[ str, str, str ]` and here a `Tuple[ str, str, str, str ]`....

Reply:
Then change whitelist in line 131 and here into a list.